content/docs/command-reference/import-url.md

```diff
@@ -132,6 +132,10 @@ $ dvc run -n download_data \
   already exist locally and you want to "DVCfy" this state of the project (see
   also `dvc commit`).
 
+- `-j <number>`, `--jobs <number>` - parallelism level for DVC to download data
```

Comment:
@isidentical just to make sure - it's not affected by any configs at all, right? (I understand that this is not about remotes in it's default mode, but we also have core.jobs or something if I remember it right)

also, what happens if we do:

`dvc import-url remote://name ` syntax and remote specifies a `jobs` in its config - is it a problem?

Reply:
If I understood https://github.com/iterative/dvc.org/pull/2122#discussion_r568840460 correctly, the configuration of the source repo is used.

Reply:
For `import-url`, it goes with `--jobs => remote.jobs => core.jobs => BaseTree.JOBS`

Reply:
`BaseTree.JOBS` is the hardcoded default right? 4

Reply:
> BaseTree.JOBS is the hardcoded default right? 4

`4 * cpu_count()`

Reply:
Also note that not `BaseTree.JOBS` but more like `Tree.JOBS`, as defaults are different for different clouds. For ssh it is 4 for the rest it is 4 * NCPU.

Reply:
does it make sense to mention this here as we do in the other parts of this PR?

Reply:
@shcheklein It is already mentioned here.

Reply:
Hmm, I don't see it. (I don't mean defaults, but rather that it's affected or can be affected by config. We have something like this in the other places)

Reply:
I see. Sure, could mention here.